dash-pipeline/bmv2/underlay.p4

```diff
@@ -33,6 +33,16 @@ control underlay(
 #endif // TARGET_DPDK_PNA  
     }
 
+    action pkt_act(bit<9> packet_action, bit<9> next_hop_id) {
+        if(packet_action == 0) {
```

Comment:
Would prefer to use same symbol names as the C code i.e. `SAI_PACKET_ACTION_FORWARD, SAI_PACKET_ACTION_DROP` and `#define` them at the top of the file, with a comment referencing the sai header files they come from. (It's not practical to include the sai header files as-is into the P4 code).

Reply:
yea working on it.